plugins/repository-s3/build.gradle

```diff
@@ -187,15 +187,30 @@ if (useFixture && minioDistribution) {
     dependsOn installMinio
 
     ext.minioPid = 0L
+    ext.minioPort = 0
 
     doLast {
+      // get free port
+      for (int port = 60920; port < 60940; port++) {
+        try {
+          javax.net.ServerSocketFactory.getDefault().createServerSocket(port, 1, InetAddress.getByName(minioAddress)).close()
+          minioPort = port
+          break
+        } catch (Exception e) {
+          logger.error("Failed to determine free port " + port + " for Minio process", e)
```

Comment:
We should not log this at error level, as the retry will solve it, also would be better to catch the specific `BindException` and just log info

Reply:
right, I've fixed this in d4add8b. Catching Exception here will possibly hide other issues.